requirements.txt

```diff
@@ -1,3 +1,3 @@
-jsonschema>=2.5.1
-xmltodict>=0.11.0
-furl>=0.5.6
+jsonschema~=4.0
+xmltodict>=0.13.0
+furl>=2.1.3
```

Comment:
@sarayourfriend  These are much too over-constraining!  I strongly recommend reverting changes in this file before making a new release of pook.

Reply:
You're right. My mind was in node dependencies land, not Python. Fixed in https://github.com/h2non/pook/pull/93.